tsconfig.json

```diff
@@ -28,5 +28,5 @@
       "@biblionexus-foundation/scripture-utilities": ["./utilities/src"]
     }
   },
-  "include": ["packages"]
+  "include": ["packages", "packages/shared/plugins/PerfOperations/epitelete.d.ts"]
```

Comment:
Is this really needed or was this just VS Code trying to be helpful when the file was moved or renamed?